### ISSUE TITLE
[ENHANCEMENT] Comment Options - Add "Full Context" as an option for the View Parent Button

### ISSUE THREAD

**Issue by lazyboy0172** [laurencedawson/sync-for-reddit#1318]
Maybe as an ultra unlock, I think a nice enhancement to the 'View Parent Button' option would be to include "full context" in addition to the current root/parent options. Not sure if this would cause issues for long threads, but I find this setting hard to choose because sometimes on a long post I just want to see the immediate parent, other times I want to see further up the chain. Having the option for the pop-up to show multiple posts from parent to root would be a very welcome addition, even as a paid feature

**Comment by laurencedawson** [comment]
Long press the view parent button

**Comment by lazyboy0172** [comment]
That's a helpful tip, but not exactly what I was meaning. That is a great way to scroll to the top of the comment chain that I didn't know about, but it's still easy to lose the context of the chain.

What I'm suggesting is similar to what we already have with the 'parent comment' shade shown here:
![image](https://user-images.githubusercontent.com/57650025/170258362-0b7e6eea-4f7c-4116-baa8-317c2b4ff86d.png)

but instead of showing 1 comment and the option being immediate parent or top-of-chain, have an option to show all 'context' of that reply. so with the same comment from the above screenshot, I'd want to see the shade with dividers and all of the 5 comments called out here: 
![image](https://user-images.githubusercontent.com/57650025/170258575-1cff5dd4-f1e4-43b6-95db-d53c6202a8a9.png)

if those 2 'other' comments weren't collapsed, I would have ~30 comments between what I'm reading "so he knocked him down" and the top, with a lot of replies to other comments, etc that make it hard to follow the "chain" of the comment I'm looking at. It would be great to have an option for that parent comment shade to show all of the singular comments preceding the comment I'm looking at which were in the chain of replies.

Like I said at the top that may get sticky if we're 10 comments deep or have very long comments, etc that make the shade very long to scroll, but hopefully that's a solvable problem
